client/src/components/Footer.js

```diff
@@ -1,34 +1,50 @@
 import React from "react";
 import "../styles/Footer.css";
 import CYFLogo from "../assets/CYF.jpeg";
-import { Divider, Grid, Image, Segment } from "semantic-ui-react";
+import {
+  Container,
+  Grid,
+  Image,
+  Segment,
+  List,
+  Header,
+  Divider
+} from "semantic-ui-react";
 
 const Footer = () => {
   return (
-    <div className="footer">
-      <Grid divided columns={3}>
-        <Grid.Column>
-          <div className="team-container">
-            <h3>Min-Imece-Team</h3>
-            <span>Samuel</span>
-            <span>Hacer</span>
-            <span>Fatma</span>
-            <span>Mawaddah</span>
-            <span>Natalia</span>
-          </div>
-        </Grid.Column>
-        <Grid.Column>
-          <p>
-            A dashboard for Maryhill Integration Network that staff can use to
-            better track, and support service users and volunteers created by
-            CodeYourFuture Students
-          </p>
-        </Grid.Column>
-        <Grid.Column>
-          <img class="footer-image" src={CYFLogo} alt="CYFLogo" />
-        </Grid.Column>
-      </Grid>
-    </div>
+    <Segment className="footer" inverted vertical textAlign="center">
+      <Container textAlign="center">
+        <Grid divided columns={3}>
+          <Grid.Column width={4}>
+            <Header inverted as="h4" content="Min-Imece-Team" />
+            <List link inverted>
+              <List.Item as="a">Samuel</List.Item>
+              <List.Item as="a">Natalia</List.Item>
+              <List.Item as="a">Mawaddah</List.Item>
+              <List.Item as="a">Fatma</List.Item>
+              <List.Item as="a">Hacer</List.Item>
+            </List>
+          </Grid.Column>
+          <Grid.Column width={8}>
+            <p>
+              A dashboard for Maryhill Integration Network that staff can use to
+              better track, and support service users and volunteers created by
+              CodeYourFuture Students
+            </p>
+          </Grid.Column>
+          <Grid.Column width={4}>
+            <Image
+              src={CYFLogo}
+              as="a"
+              size="tiny"
+              href="https://codeyourfuture.io"
+              target="CYF"
```

Comment:
We are using `target` incorrectly here. Usually, it's used to tell the browser to open the link in a new window but since we don't need to do that let's remove `target`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target